docker/images/skycoindev/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# skycoin build
+# reference https://github.com/skycoin/skycoin
+FROM golang:1.10.0-stretch
+
+# dirs
+RUN apt-get update 
+RUN apt-get install -y cmake gdbserver gdb vim ctags vim-scripts screen
+RUN git clone --recurse-submodules -j8 https://github.com/Snaipe/Criterion
+RUN cd /go/Criterion && cmake .
+RUN cd /go/Criterion && make install
+RUN go get -u github.com/derekparker/delve/cmd/dlv
+
+WORKDIR $GOPATH/src/github.com/skycoin/skycoin
+VOLUME $GOPATH/src/
+
+ENV LD_LIBRARY_PATH=/usr/local/lib
```

Comment:
Maybe a bit more paths should be added in here , and other env vars needed for compilation and running Makefile .